packages/core/src/contrib/Queries.ts

```diff
@@ -409,14 +492,46 @@ export const shapeDistanceRectLine = (
   const halfH = div(bbox.height, 2);
   const [cx, cy] = bbox.center;
   return rectLineDist(
-    {
-      bottomLeft: [sub(cx, halfW), sub(cy, halfH)],
-      topRight: [add(cx, halfW), add(cy, halfH)],
-    },
-    { start, end }
+    sub(cx, halfW),
+    sub(cy, halfH),
+    add(cx, halfW),
+    add(cy, halfH),
+    start[0],
+    start[1],
+    end[0],
+    end[1]
   );
 };
 
+const shapeDistanceRectlikePolyline = (
```

Comment:
For these branches of `shapeDistance`, is it possible to include these into `compDict` as well? For example, `shapeDistanceCircleLine` is in `Queries.ts`, but we also add this into `compDict` so that the Style DSL can call this function.

Reply:
Sure thing.